### PR TITLE
Use our own client library instead of vendoring from knative.dev/serving

### LINKED ISSUE #51: Stop using Istio client library from knative.dev/serving
Now that we separated the repo, we should start generating the client library in our repo instead of relying on vendoring from knative.dev/serving

### PR DESCRIPTION
**Pull request by tcnghia**
Fix #51

### CODE REVIEW

File: pkg/reconciler/accessor/istio/virtualservice_test.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"knative.dev/pkg/controller"
 	"knative.dev/pkg/ptr"
```
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/accessor/istio/virtualservice_test.go [hunk 2]
```diff
@@ -29,11 +29,11 @@ import (
...
-	istioclientset "knative.dev/serving/pkg/client/istio/clientset/versioned"
-	istiofake "knative.dev/serving/pkg/client/istio/clientset/versioned/fake"
-	istioinformers "knative.dev/serving/pkg/client/istio/informers/externalversions"
-	fakeistioclient "knative.dev/serving/pkg/client/istio/injection/client/fake"
-	istiolisters "knative.dev/serving/pkg/client/istio/listers/networking/v1alpha3"
+	istioclientset "knative.dev/net-istio/pkg/client/istio/clientset/versioned"
+	istiofake "knative.dev/net-istio/pkg/client/istio/clientset/versioned/fake"
+	istioinformers "knative.dev/net-istio/pkg/client/istio/informers/externalversions"
+	fakeistioclient "knative.dev/net-istio/pkg/client/istio/injection/client/fake"
+	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
```
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/pkg/controller"
	"knative.dev/pkg/ptr"
```

File: pkg/reconciler/ingress/lister.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ import (
 	corev1listers "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/tools/cache"
 	"knative.dev/serving/pkg/apis/networking/v1alpha1"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go [hunk 4]
```diff
@@ -30,7 +30,7 @@ import (
...
-	istiolisters "knative.dev/serving/pkg/client/istio/listers/networking/v1alpha3"
+	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/serving/pkg/apis/networking/v1alpha1"
```

File: pkg/reconciler/accessor/istio/virtualservice_test.go (hunk 1 above)
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/accessor/istio/virtualservice_test.go (hunk 2 above)
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/pkg/controller"
	"knative.dev/pkg/ptr"
```

File: pkg/reconciler/ingress/lister.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/serving/pkg/apis/networking/v1alpha1"
```

File: pkg/reconciler/accessor/istio/virtualservice_test.go (hunk 1 above)
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/accessor/istio/virtualservice_test.go (hunk 2 above)
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/pkg/controller"
	"knative.dev/pkg/ptr"
```

File: pkg/reconciler/ingress/lister.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/serving/pkg/apis/networking/v1alpha1"
```

File: pkg/reconciler/accessor/istio/virtualservice_test.go (hunk 1 above)
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/accessor/istio/virtualservice_test.go (hunk 2 above)
Comment (mattmoor) [inline pkg/reconciler/accessor/istio/virtualservice_test.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/pkg/controller"
	"knative.dev/pkg/ptr"
```

File: pkg/reconciler/ingress/lister.go (hunk 3 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 4 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	istiolisters "knative.dev/net-istio/pkg/client/istio/listers/networking/v1alpha3"
	"knative.dev/serving/pkg/apis/networking/v1alpha1"
```

File: test/e2e/istio/probing_test.go [hunk 5]
```diff
@@ -1,7 +1,7 @@
 // +build e2e istio
 
 /*
-Copyright 2020 The Knative Authors
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/e2e/istio/probing_test.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

**Comment by tcnghia** [comment]
/assign @ZhiminXiang 
cc @trshafer

File: pkg/reconciler/testing/listers.go [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+// IndexerFor returns the indexer for the given object.
+func (l *Listers) IndexerFor(obj runtime.Object) cache.Indexer {
+	return l.sorter.IndexerForObjectType(obj)
+}
+
+func (l *Listers) GetServingObjects() []runtime.Object {
+	return l.sorter.ObjectsForSchemeFunc(fakeservingclientset.AddToScheme)
+}
```
Comment (mattmoor) [inline pkg/reconciler/testing/listers.go]: I don't think you need this

Comment (tcnghia) [reply pkg/reconciler/testing/listers.go]: done

**Pull request merged**